src/main/java/org/opentripplanner/hasura_client/BikeStationsGetter.java

```diff
@@ -0,0 +1,23 @@
+package org.opentripplanner.hasura_client;
+
+import org.opentripplanner.hasura_client.hasura_objects.BikeStationHasura;
+import org.opentripplanner.hasura_client.mappers.BikeStationsMapper;
+import org.opentripplanner.hasura_client.mappers.HasuraToOTPMapper;
+import org.opentripplanner.routing.bike_rental.BikeRentalStation;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class BikeStationsGetter extends HasuraGetter<BikeRentalStation, BikeStationHasura> {
```

Comment:
I've decided to add this class in this PR to show how new downloading simplifies fetching new info from hasura